plugins/aea-ledger-solana/aea_ledger_solana/solana.py

```diff
@@ -64,6 +64,9 @@
 from aea.helpers.base import try_decorator
 
 
+DEFAULT_MAX_SUPPORTED_TRANSACTION_VERSION = 100
```

Comment:
@8ball030 @Dassy23 Is this correct for default max supported transaction version?

Reply:
No I think it should be 0
Refer to https://solanacookbook.com/guides/versioned-transactions.html#rpc-changes

Reply:
Addressed https://github.com/valory-xyz/open-aea/pull/699/commits/e0bc608634c69c88bbb78372133ec9d458266d51